.gitignore

```diff
@@ -13,3 +13,4 @@ htmlcov/
 venv/
 .idea/
 .tox/
+uv.lock
```

Comment:
This should be in your global gitignore file, not in every git ignore in every repositories you ever interact with :)

Reply:
Well, some projects do commit the lock file, so global git ignore is the right place either. But I didn't meant to include it in this pull request.